src/models/bundle-descriptor-constraints.ts

```diff
@@ -38,7 +38,7 @@ import {
 export const ALLOWED_NAME_REGEXP = /^[\da-z]+(?:([.-])[\da-z]+)*$/
 export const ALLOWED_VERSION_REGEXP = /^\w+[\w.-]*$/
 export const MAX_VERSION_LENGTH = 128
-export const MAX_NAME_LENGTH = 50
+export const MAX_NAME_LENGTH = 31
```

Comment:
Hi @TheLearneer  ,
that variable is used to validate a lot of content types' length. The issue you are talking about seems to be limited to the Mfe name, can we ask you to add another const to use only to validate Mfe name length?
It should be the line 
`if (mfe.name.length > MAX_NAME_LENGTH) {`
within `src/services/microfrontend-service.ts`
If you are busy we can easily proceed by ourselves, just let us know please

Reply:
@firegloves I can proceed with the fix.
Just to be sure though, I believe this fix should be done for both microfrontend and microservice, right?

Reply:
Thanks @TheLearneer, I don't think so because the limit to 40 chars takes place within the Entando core database. Microservice are handled by a different Entando component and the main storage is directly kubernetes, so I'm pretty sure they have different limits. Did you experienced something different?